### DISCUSSION TITLE
How to extract an image from a pdf

### DISCUSSION THREAD

**Discussion by JC-Yodiwo** [QuestPDF/QuestPDF#693] [General]
Given a pdf which contains an image, how to extract this image with its actual width and height

**Comment by girlpunk** [comment]
QuestPdf doesn't currently have any support for opening or editing existing PDFs. You may be able to achieve this with other libraries such as [PdfPig](https://github.com/UglyToad/PdfPig)

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. As @girlpunk mentioned, QuestPDF is a PDF generation library and does not support reading or extracting content from existing PDFs. Libraries such as PdfPig would be more suited for this use case.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
